src/awesome_map/EventSynthesizer.js

```diff
@@ -460,6 +483,17 @@ define(function(require) {
             };
         },
 
+        _getMsHoldVisualHandler: function() {
+            var self = this;
+
+            return function(event) {
+                if (self._cancelTouchContextMenuEvents) {
+                    // Prevent the Windows "box hint" that appears during a hold.
+                    event.preventDefault();
+                }
+            };
```

Comment:
Maybe this should be (yet another) configuration option that defaults to true. Call it `cancelMsHoldVisual`...